backend/canisters/user/api/can.did

```diff
@@ -225,6 +225,15 @@ type UnblockUserResponse = variant {
     UserSuspended;
 };
 
+type DeleteDirectChatArgs = record {
+    user_id : UserId;
```

Comment:
Perhaps it should take a block: bool too?

Reply:
Done